<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>club_information</class>
 <widget class="QMainWindow" name="club_information">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1038</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Club Information</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="club_name_label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>40</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Club Name:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="club_combo_box">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>80</y>
      <width>381</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <item>
     <property name="text">
      <string>SerVe</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="team_label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>170</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Team:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="team_combo_box">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>210</y>
      <width>381</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <item>
     <property name="text">
      <string>Design</string>
     </property>
    </item>
   </widget>
   <widget class="QGroupBox" name="member_group_box">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>280</y>
      <width>381</width>
      <height>291</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Members</string>
    </property>
    <widget class="QTableWidget" name="members_table">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>361</width>
       <height>192</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <row>
      <property name="text">
       <string>0</string>
      </property>
     </row>
     <column>
      <property name="text">
       <string>Student ID</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Name</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Position</string>
      </property>
     </column>
     <item row="0" column="0">
      <property name="text">
       <string>jd0669</string>
      </property>
     </item>
     <item row="0" column="1">
      <property name="text">
       <string>John Doe</string>
      </property>
     </item>
     <item row="0" column="2">
      <property name="text">
       <string>Member</string>
      </property>
     </item>
    </widget>
    <widget class="QPushButton" name="add_member_btn">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>240</y>
       <width>101</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Add</string>
     </property>
    </widget>
    <widget class="QPushButton" name="add_member_btn_2">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>240</y>
       <width>101</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>View</string>
     </property>
    </widget>
    <widget class="QPushButton" name="remove_member_btn">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>240</y>
       <width>101</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Remove</string>
     </property>
    </widget>
   </widget>
   <widget class="QTextEdit" name="funds_text_edit">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>610</y>
      <width>241</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:12pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;750,000&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="funds_label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>580</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Funds:</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="resources_borrowed_group_box">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>30</y>
      <width>551</width>
      <height>281</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Resources borrowed</string>
    </property>
    <widget class="QTableWidget" name="resources_borrowed_tbl">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>531</width>
       <height>241</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <row>
      <property name="text">
       <string>0</string>
      </property>
     </row>
     <column>
      <property name="text">
       <string>New Column</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Resource ID</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Event</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Due Date</string>
      </property>
     </column>
     <item row="0" column="0">
      <property name="text">
       <string>MIC001</string>
      </property>
     </item>
     <item row="0" column="1">
      <property name="text">
       <string>Microphone</string>
      </property>
     </item>
     <item row="0" column="2">
      <property name="text">
       <string>DiscO</string>
      </property>
     </item>
     <item row="0" column="3">
      <property name="text">
       <string>24/09/2023</string>
      </property>
     </item>
    </widget>
   </widget>
   <widget class="QGroupBox" name="events_planned_group_box">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>320</y>
      <width>551</width>
      <height>281</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Events planned</string>
    </property>
    <widget class="QTableWidget" name="events_planned_tbl">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>531</width>
       <height>241</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
     <row>
      <property name="text">
       <string>0</string>
      </property>
     </row>
     <column>
      <property name="text">
       <string>Event Name</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Event Date</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Event Time</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Location</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Feedback</string>
      </property>
     </column>
     <item row="0" column="0">
      <property name="text">
       <string>DiscO</string>
      </property>
     </item>
     <item row="0" column="1">
      <property name="text">
       <string>23/09/2023</string>
      </property>
     </item>
     <item row="0" column="2">
      <property name="text">
       <string>7:00 pm</string>
      </property>
     </item>
     <item row="0" column="3">
      <property name="text">
       <string>Earth Courtyard</string>
      </property>
     </item>
     <item row="0" column="4">
      <property name="text">
       <string>The event went smoothly as we secured one of the most</string>
      </property>
     </item>
    </widget>
   </widget>
   <widget class="QPushButton" name="update_funds_btn">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>610</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Update</string>
    </property>
   </widget>
   <widget class="QPushButton" name="close_btn">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>610</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Close</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1038</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
